book/tutorials_examples/1_HBV_Caravan_ERA5/example_model_run_HBV.ipynb

```diff
@@ -34,7 +34,7 @@
     "The stucture of the HBV model is shown below.\n",
```

Comment:
<p>shall we make the paths more general? tutorials or something? So we can re-use this for different workshops?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/projects/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>